src/core/Folder.java

```diff
@@ -0,0 +1,33 @@
+package core;
+
+import java.util.List;
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ */
+public class Folder extends Item {
+
+    public Folder (File file) {
+        super(file);
+    }
+
+    /**
+     * List of files and folders that are on a path
+     */
+    public List<String> listFilesAndFolders(String directoryName)
+    {
+        List<String> fileListed = new ArrayList<String>();
+        File directory = new File(directoryName);
```

Comment:
What happen if the directory does not exists? usually when you are doing operation with files or directories it is a good practice to handle exceptions